includes/Helpers/Plugin.php

```diff
@@ -87,4 +91,29 @@ public static function does_it_autoupdate( $slug ) {
 
 		return in_array( $slug, $wp_auto_updates, true );
 	}
+
+	/**
+	 * Get Admin and SuperAdmin user accounts
+	 *
+	 * @return $users Array of Admin & Super Admin users
+	 */
+	private static function get_admin_users() {
```

Comment:
* We should generally prefer non-static functions. A couple of reasons (although not relevant here): they're easier to work with when testing and easier to work with when extending classes.
* Please include the return type `: array`